src/kem/kyber/pqclean_kyber1024_aarch64/fips202x2.h

```diff
@@ -1,10 +1,17 @@
+
+/*
+ * This file is licensed
+ * under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.html)
```

Comment:
Another license for the YML...

Reply:
I read here Apache-2.0 or CC0 (for the SPDX-id)